packages/celotool/src/cmds/unfreeze_contracts.ts

```diff
@@ -0,0 +1,114 @@
+/* tslint:disable no-console */
+import { newKit } from '@celo/contractkit'
+import { eqAddress } from '@celo/utils/lib/address'
+import { switchToClusterFromEnv } from 'src/lib/cluster'
+import { addCeloEnvMiddleware, CeloEnvArgv } from 'src/lib/env-utils'
+import { portForwardAnd } from 'src/lib/port_forward'
+import yargs from 'yargs'
+
+export const command = 'unfreeze-contracts'
+
+export const describe = 'command for unfreezing epoch rewards and the exchange'
+
+interface UnfreezeContractsArgv extends CeloEnvArgv {
+  exchange: boolean
+  rewards: boolean
+  freeze: boolean
+  precheck: boolean
+  verify: boolean
+}
+
+export const builder = (argv: yargs.Argv) => {
+  return addCeloEnvMiddleware(argv)
+    .option('exchange', {
+      type: 'boolean',
+      description: 'Affect the exchange',
+      default: true,
+    })
+    .option('rewards', {
+      type: 'boolean',
+      description: 'Affect epoch rewards',
+      default: true,
+    })
+    .option('freeze', {
+      type: 'boolean',
+      description: 'Freeze contracts instead of unfreezing',
+      default: false,
+    })
+    .option('precheck', {
```

Comment:
Shouldn't we always do this?

Reply:
I want this to be an option in case it causes some kind of issue.